BLSQ/iaso
# IA-2972: Double check the mobile feature flags descriptions

Explain what problem this PR is resolving
- Double check the mobile feature flags descriptions
- Have mobile feature flags available in French
Related JIRA tickets : [IA-2972](https://bluesquare.atlassian.net/browse/IA-2972)
## Self proofreading checklist


## Changes
- Add french translations for featureFlags
- Fix tooltip description

## How to test

- Go in projects
- Change the language to french
- Check if featureFlags descriptions are correct

[IA-2972]: https://bluesquare.atlassian.net/browse/IA-2972?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
